cves/CVE-2014-1418.yml

```diff
@@ -274,7 +309,15 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
At what stage do you think this was this failure occurred or do you think it was just a Coding Mistake?